.github/workflows/test.yml

```diff
@@ -0,0 +1,101 @@
+name: Test
+
+on:
+  # Run on all pushes and on all pull requests.
+  # Prevent the "push" build from running when there are only irrelevant changes.
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      # Keys:
+      # - php: The PHP versions to test against.
+      # - phpcs_version: The PHPCS versions to test against.
+      #   IMPORTANT: test runs shouldn't fail because of PHPCS being incompatible with a PHP version.
+      #   - PHPCS will run without errors on PHP 5.4 - 7.4 on any supported version.
+      #   - PHP 8.0 needs PHPCS 3.5.7+ to run without errors.
+      # - The `wpcs_version` key is added to allow additional test builds when multiple WPCS versions
+      #   would be supported. As, at this time, only the latest stable release of WPCS is supported,
+      #   no additional versions are included in the array.
+      # - experimental: Whether the build is "allowed to fail".
+      matrix:
+        php: ['5.4', '5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4']
+        phpcs_version: ['3.5.5', 'dev-master']
+        wpcs_version: ['2.3.*']
+        experimental: [false]
+
+        include:
+          # Complete the matrix by adding PHP 8.0, but only test against compatible PHPCS versions.
+          - php: '8.0'
+            phpcs_version: 'dev-master'
+            wpcs_version: '2.3.*'
+            experimental: false
+          - php: '8.0'
+            # PHPCS 3.5.7 is the lowest version of PHPCS which supports PHP 8.0.
+            phpcs_version: '3.5.7'
+            wpcs_version: '2.3.*'
+            experimental: false
+
+          # Experimental builds. These are allowed to fail.
+          #- php: '8.1'
+          #  phpcs_version: 'dev-master'
+          #  wpcs_version: '2.3.*'
+          #  experimental: true
+
+    name: "Test: PHP ${{ matrix.php }} - PHPCS ${{ matrix.phpcs_version }} - WPCS ${{ matrix.wpcs_version }}"
```

Comment:
```suggestion
    name: "PHP ${{ matrix.php }} - PHPCS ${{ matrix.phpcs_version }} - WPCS ${{ matrix.wpcs_version }}"
```

Each line is already prefixed with `Test` from line 1, so it seems redundant to include it again here.

<img width="928" alt="Screenshot 2021-03-04 at 11 17 45" src="https://user-images.githubusercontent.com/88371/109956244-56bf6d00-7cdb-11eb-9558-6f1ecef9f7f6.png">

Reply:
> Each line is already prefixed with Test from line 1, so it seems redundant to include it again here.

I realize it seems that way, however, there is a reason for it.

In the PR block below, each check is shown in the form "workflow name/job name/trigger event". This includes ALL checks, i.e. both those done on `push` as well as those run on `pull_request`. So yes, here it is easy to see what's what and the prefix looks redundant.

However, now, check in the "Settings" -> "Branches" -> protected branch/required statuses.

There you will _only_ see the "job name" to choose which jobs to set as required. For now, that is not an issue, but as soon as I would add the "Quick test" workflow as is being discussed in #628, you would no longer be able to tell which runs are from the "Quick test" job (not required) and which ones are from "Test" job (required).

Does that make it clearer why I prefixed the job names ?